stellargraph/core/convert.py

```diff
@@ -134,6 +141,26 @@ def select_column(old_name):
 
         return ids, columns, features
 
+    def _convert_rowframe(self, type_name, data):
+        assert isinstance(data, (RowFrame, np.ndarray))
+        if self.selected_columns:
+            raise ValueError(
+                f"{self.name(type_name)}: expected a Pandas DataFrame when selecting columns {comma_sep(self.selected_columns)}, found {type(data).__name__}"
```

Comment:
I think this error will be raised when constructing from a `RowFrame` of edges

Reply:
Yes, that's the intention; what you getting at here?

Reply:
just referencing the first comment above ^

Reply:
Ah, right.